src/org-manager/containers/active-organisations/active-organisations.component.ts

```diff
@@ -24,7 +25,7 @@ export class ActiveOrganisationsComponent extends OrganisationListComponent {
     return fromStore.getActiveOrganisationArray;
   }
 
-  constructor(protected readonly store: Store<fromStore.OrganisationRootState>) {
-    super(store);
+  constructor(protected readonly store: Store<fromStore.OrganisationRootState>, route: ActivatedRoute) {
```

Comment:
Is route needed in this component? If so, it probably needs to be marked as private readonly. Additionally, what is the reason for putting it in super()?

Reply:
it needs to pass to superclass cant add private but we can add protected instead of private

Reply:
I see you've changed it from private to protected readonly. Resolved.